src/webrtc/call.ts

```diff
@@ -828,9 +849,16 @@ export class MatrixCall extends TypedEventEmitter<CallEvent, CallEventHandlerMap
 
                     // this is what would allow us to use addTransceiver(), but it's not available
                     // on Firefox yet. We call it anyway if we have it.
-                    if (transceiver.sender.setStreams) transceiver.sender.setStreams(callFeed.stream);
+                    if (transceiver.sender.setStreams) transceiver.sender.setStreams(callFeed.stream!);
```

Comment:
Comment again suggests this will break if we switch to addTransceiver, but we have done - will this work?